utils/download_figshare.py

```diff
@@ -61,3 +59,27 @@ def download_figshare(
         # remove the metadata from the plate directory to make it an images only directory
         os.remove(files)
     print("The metadata has been moved into its own directory!")
+
+
+def extract_zip_from_Figshare(
```

Comment:
If it makes sense, consider returning the unzipped filepath to enable functional programming with this work.

Reply:
I think the current iteration of this function is fine without the return.